src/cloudant/design_document.py

```diff
@@ -153,6 +209,18 @@ def fetch(self):
                         view_def.pop('reduce', None),
                         **view_def
                     )
+        if not self.search_indexes:
+            # Ensure indexes dict exists in locally cached DesignDocument.
+            self.setdefault('indexes', dict())
+        if self.search_indexes:
+            for (search_index_name, search_index) \
+                    in iteritems_(self.get('indexes', dict())):
```

Comment:
Lets work on getting this on one line.  There really shouldn't be a reason for this for statement have to span multiple lines.  Consider shortening the names of the elements in the tuple and perhaps perform the `iteritems_` call or the `self.get(...)` call on a separate line but the `for` logic should really be on one line.